### DISCUSSION TITLE
[Feature] Adminstration - Repair - indicate, what happens and that something happens

### DISCUSSION THREAD

**Discussion by aisbergde** [immich-app/immich#6065] [Feature Request]
### The feature

When I click Administration - Repair, then several minutes, nothing happens. This is irritating. Some information would be nice, that something is happening in the background and what is happening. Otherwise, the user could just think, that this is a bug. 

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by danieldietzler** [comment]
We are aware of that issue and generally we want to refactor this so that it won't load when you access the data but beforehand. Currently, some people can't access the page at all, even with waiting a long time, because the request times out. So I don't think a loading indicator is the solution but rather making the page usable by making it a lot faster is.
